src/shared/molecules/AdvancedMode/AdvancedMode.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { useLocation, useHistory } from 'react-router';
+import { match as pmatch } from 'ts-pattern';
+import { useSelector, useDispatch } from 'react-redux';
+import { Switch } from 'antd';
+import { RootState } from '../../store/reducers';
+import { UISettingsActionTypes } from '../../store/actions/ui-settings';
+import './styles.less';
+
+export const advancedModeBlackList = ['/studios', '/studio', '/data-explorer'];
```

Comment:
According to the figma file the advanced toggle should still be visible in the `/data-explorer` route. (I've already made this change in PR [1285](https://github.com/BlueBrain/nexus-web/pull/1285) so feel free to ignore this).